RNReanimated.podspec

```diff
@@ -87,6 +88,9 @@ Pod::Spec.new do |s|
   s.dependency 'Yoga'
   s.dependency 'DoubleConversion'
   s.dependency 'glog'
+  if config[:react_native_minor_version] >= 65 && use_hermes
```

Comment:
Can you please remind me why `config[:react_native_minor_version] >= 65` is necessary here?

Reply:
Becuase it doesn't build otherwise 🤷 
It can't find the `React-hermes` pod in that case (the pod was called `React-Core/Hermes` back then).

Reply:
What about this?
```rb
  if use_hermes
    if config[:react_native_minor_version] >= 65
      s.dependency "React-hermes"
    else
      s.dependency "React-Core/Hermes"
    end
  end
```

Reply:
It builds fine on 0.64 without this dependency, so I'd leave it as is :+1: